nebu/cli/validate.py

```diff
@@ -23,10 +23,8 @@ def is_valid(struct):
         try:
             filepath = filepath.relative_to(cwd)
         except ValueError as err:
-            if 'does not start with' in str(err):
-                pass
-            else:
-                raise
+            # Raised when filepath is not a child of cwd
+            pass
```

Comment:
I do not know if there is anything else we could do besides `pass` it I really hope we do and if so, I'd like to suggest that change :).

Reply:
Nope, the underlying library raises this under these specific conditions, and only this one. If this is true, then we have to give the fullpath.